Makefile

```diff
@@ -0,0 +1,27 @@
+
+.DEFAULT_GOAL := help
+
+.PHONY: help
+help: ## Show this help.
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+node_modules: ## reinstall sass dependencies
+	@which -s npm || (echo 'npm is required to install the upstream sass dependencies. Install node js + npm.' && false)
+	npm install
+
+vendor: # reinstall ruby dependencies
```

Comment:
You should run `make clean` if the target is stale. Making a target phony would make it run on every invocation.

Reply:
We've talked past each other a little. The reason this came up is that running `make vendor node_modules` on a fresh checkout doesn't do anything, because `vendor` and `node_modules` are checked in to the repo and thus already exist. Their contents are already ignored, so removing them from the repo (run `make clean` and commit then) would help?

Reply:
With @chrisfarms away today, I'll resolve this so we can get this into approving.

Reply:
Hmm, that was already no longer true that `vendor` and node_modules are checked in. Approving.